activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -276,7 +277,13 @@ def _active_record_fixture(fixture_set_name, *fixture_names)
           raise StandardError, "No fixture set named '#{fixture_set_name.inspect}'"
         end
       end
-      alias_method :fixture, :_active_record_fixture
+
+      # Generic fixture accessor for fixture names that may conflict.
+      #
+      #   assert_equal "Ruby on Rails", web_sites(:rubyonrails).name
+      #   assert_equal "Ruby on Rails", fixture(:web_sites, :rubyonrails).name
+      public def fixture(fixture_set_name, *fixture_names); end
+      silence_warnings { alias_method :fixture, :_active_record_fixture }
```

Comment:
```suggestion
      silence_redefinition_of_method(:fixture)
```

Reply:
Whoops, that doesn't actually alias to `_active_record_fixture` ... I got trigger happy.